graphs/adapter-jgrapht/src/main/kotlin/com/anaplan/engineering/azuki/graphs/adapter/jgrapht/check/HasShortestPathCheck.kt

```diff
@@ -10,20 +10,20 @@ import org.jgrapht.graph.DefaultEdge
 
 class HasShortestPathCheck<V>(
     private val graphName: String,
-    private val shortestPath: List<V>,
+    private val path: List<V>,
 ) : JGraphTCheck, GetShortestPathBehaviour() {
 
     init {
-        if (shortestPath.size < 2) {
-            throw LateDetectUnsupportedCheckException("")
+        if (path.size < 2) {
+            throw LateDetectUnsupportedCheckException("hasShortestPath must take a start and end vertex in the path, as inputs")
```

Comment:
🚫 This reads a bit weird. I would say something like "path must contain at least its two endpoints".

 P.S.: This _really_ shouldn't make any reference at all to a DSL function name, because they belong to two completely separate layers.